packages/compartment-mapper/test/test-main.js

```diff
@@ -227,7 +227,21 @@ scaffold(
 );
 
 scaffold(
-  'package root as main via internalAlias',
+  'package root as main via internalAlias (.)',
+  test,
+  new URL(
+    'fixtures-0/node_modules/excalibur/sword.js',
+    import.meta.url,
+  ).toString(),
+  (t, { namespace: { sword } }) => {
+    t.is(typeof sword, 'string');
+    t.is(sword.length, 30);
+  },
+  2,
+);
+
+scaffold(
+  'package root as main via internalAlias (..)',
```

Comment:
this path (`..`) first gets resolved to package root (`.`) before searching for candidates, so these tests should be equivalent